base58/Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[package]
+name = "bitcoin-base58"
+version = "0.1.0"
+authors = ["Andrew Poelstra <apoelstra@wpsoftware.net>", "The Rust Bitcoin developers"]
+license = "CC0-1.0"
+repository = "https://github.com/rust-bitcoin/rust-bitcoin/"
+documentation = "https://docs.rs/bitcoin-base58"
+description = "Bitcoin base58 encoding format"
+categories = ["cryptography::cryptocurrencies"]
+keywords = ["bitcoin", "encoding"]
+readme = "README.md"
+edition = "2018"
+
+# Please don't forget to add relevant features to docs.rs below.
+[features]
+default = ["std"]
+std = ["bitcoin_hashes/std"]
+alloc = ["bitcoin_hashes/alloc"]
+
+[package.metadata.docs.rs]
+features = ["std"]
+rustdoc-args = ["--cfg", "docsrs"]
+
+[dependencies]
+bitcoin_hashes = { version = "0.11.0", default-features = false }
```

Comment:
Would it be worth it to make this optional? IOW, is there any crate that doesn't need base58check and only depends on `bitcoin_hashes` via this crate?

Reply:
It's hard for me to imagine any use for `base58` without `base58check` ... but I suppose there's no *harm* in making it optional.

Reply:
Maybe if you were just converting from base58 into a less ridiculous format you could decode, drop the checksum assuming it's ok, and then convert the rest.

Reply:
>  no harm

Could be annoying for `no_std` users. But also a bloated binary could be annoying...

Reply:
Yeah, that's true... I've started habitually adding `default-features = false` to everything, even as a non-`no_std` user, and I'd be a little annoyed if I lost core functionality, that didn't even require std, because of it.